eng/testing/performance/android_scenarios.proj

```diff
@@ -1,4 +1,9 @@
 <Project Sdk="Microsoft.DotNet.Helix.Sdk" DefaultTargets="Test">
+  <PropertyGroup>
+      <IncludeXHarnessCli>true</IncludeXHarnessCli>
+      <MicrosoftDotNetXHarnessCLIVersion>1.0.0-prerelease.21566.2</MicrosoftDotNetXHarnessCLIVersion>
+      <XharnessPath>%HELIX_CORRELATION_PAYLOAD%\microsoft.dotnet.xharness.cli\$(MicrosoftDotNetXHarnessCLIVersion)\tools\net6.0\any\Microsoft.DotNet.XHarness.CLI.dll</XharnessPath>
```

Comment:
This path has the TFM net6.0 in it. Do we know where that information comes from, and when we will need to update it?

Reply:
You shouldn't need to set this. There will be a variable `%XHARNESS_CLI_PATH%` set for you with this path if you need it on the Helix machine so you can just use that one.

Once you start calling XHarness the right way (we agreed with Parker that this solution is temporary) via the [XHarness Helix SDK](https://github.com/dotnet/arcade/blob/main/src/Microsoft.DotNet.Helix/Sdk/tools/xharness-runner/Readme.md), you can just call xharness via `xharness android ...` and won't need this path in your scripts at all.